### PR TITLE
Login with email and pswd and google provider

### PR DESCRIPTION
**Pull request by goncalodasilva**

### CODE REVIEW

File: src/common/constants.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const SessionStorage = {
+    "USER_CREDENTIALS": "hom_user"
```
Comment (jocarino) [inline src/common/constants.ts:2]: I think this should be an env var maybe? pulled from a .env file?

File: src/login/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        // The AuthCredential type that was used.
+        const credential = GoogleAuthProvider.credentialFromError(error);
+        // ...
+        alert(errorMessage)
+      });
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleEmailLogin}>
```
Comment (jocarino) [inline src/login/Login.tsx:60]: image
we probaly need some minor validation here,  should be something really simple for now

Comment (jocarino) [reply src/login/Login.tsx:60]: image
how does one register with email?

Comment (goncalodasilva) [reply src/login/Login.tsx:60]: Implemento nesta task https://www.notion.so/joaoveloso/Sign-Up-with-email-9d1e299bc0d94b7ca32d5f1ec308b262?pvs=4

File: src/common/constants.ts (hunk 1 above)
Comment (goncalodasilva) [reply src/common/constants.ts:2]: I'll do that tonight

**Review by jocarino — APPROVED** [review]

**Pull request merged**